### DISCUSSION TITLE
Assignment Settings (Edit Assignment) 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by smbowers** [community/community#117369] [GitHub Education]
### Hi everyone,

I'm getting an internal server error when I try to edit an assignment. Something like this has happened in the past, and it seemed to resolve after logging out and logging back in, etc. However, I've tried all sorts of things (shut down chrome, tried firefox, incognito mode, cleared cache, etc.) and I can't get anything except a 500 Internal Server Error. 

![Screenshot from 2024-04-06 15-25-38](https://github.com/community/community/assets/641260/0379d460-c18a-4ce8-a8dd-3009e7310e99)

**Comment by jheinzel** [comment]
When trying to delete an assignment, one gets a 406 (Request Not Acceptable). Seems to be related to the issue described above.

![image](https://github.com/community/community/assets/15673869/48753968-6f80-4a00-84f8-7b38ab4ca291)

**Comment by smbowers** [comment]
I was able to edit an assignment, but when I got to the end, it saying there is an error. It is now creating multiple assignments with the same name. Also, now when I add an assignment, students aren't able to accept them -- it says they are no longer available. The whole thing seems hosed.

**Comment by smbowers** [comment]
Here are the errors I am now getting when trying to remove an assignment in classroom (that doesn't have any repos because it is not allowing anyone to accept the assignment). 

![image](https://github.com/community/community/assets/641260/faf50acd-4715-4ff8-961d-df96c5305ac2)

When I tried to edit the assignment (following the instructions given when the assignment couldn't be accepted), it created a duplicate assignment after giving an error. Here is the duplicate assignment: 

![image](https://github.com/community/community/assets/641260/733402e6-12dc-4d77-8567-6bc2d56ea817)

Here is the error I am receiving after I try to edit the assignment: 

![image](https://github.com/community/community/assets/641260/3b5bc80e-02d3-4d1a-bd56-8d632b78bf2f)

You can't make this stuff up!

**Reply by CodyHenrichsen-CTEC** [reply]
I also cannot delete assignments or have students accept them!

**Comment by samus-aran** [comment]
Hi @smbowers & @CodyHenrichsen-CTEC 👋 

Just want to ask on this set up have you opted in to the forked classroom feature?

**Reply by smbowers** [reply]
No, I haven't.  It seems like some of the above may have been resolved -- but that the classroom site/code seems to be under ongoing changes/maintenance. I was using the classroom site recently and it looks like even the styling has completely changed :-)

**Reply by CodyHenrichsen-CTEC** [reply]
Not yet @samus-aran - I was waiting until next year since there is only one more assignment. I just tried remaking it again and was able to have students link their repos to both the final and the remade project. I still cannot delete the problem assignments but that really isn't a problem anymore. I am looking forward to using the fork option next semester.

Thank you

**Reply by CodyHenrichsen-CTEC** [reply]
@samus-aran I was able to edit the project , rename the project, then delete them. Not sure what the problem was.

**Reply by samus-aran** [reply]
Thank you all for responding. We're happy you are able to make progress with assignments. 🎉 

We'll be closing this discussion as resolved but please do continue to raise any more questions or bug reports if and when they occur 👍
